addons/controls/src/components/ControlsPanel.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from 'react';
```

Comment:
Question: do we want to have a `disable` behavior for this as well?

Reply:
Probably! Will add...